terraform/templates/tile_cache_bucket_policy.json.tmpl

```diff
@@ -0,0 +1,20 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:PutLifecycleConfiguration"
```

Comment:
This should be:

"s3:PutBucketLifecycleConfiguration"

Reply:
Good catch, thanks!
I wonder why it didn't complain?